docs/content/configuration/basics.md

```diff
@@ -0,0 +1,650 @@
+
+```
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends default-mysql-client
+```
+:::
+
+:::warning
+Dump and restore is not supported yet for SQL Server.
+:::
+
+### Default binary options
+
+Apart from the essential options (credentials, database name, etc.), the bundle
+also passes a few default options to the binary depending on the operation being
+performed and the invoked binary itself. You can customize those default options
+by configuring your own ones per operation type and connection:
+
+<div class="symfony">
+
+Here is an example that sets options for all connections:
+
+```yml
+# config/packages/db_tools.yaml
+db_tools:
+    backup_options: '--an-option'
+    restore_options: '--a-first-one --a-second-one'
+```
+
+Or set a specific value list for each connection:
+
+```yml
+# config/packages/db_tools.yaml
+db_tools:
+    connections:
+        connection_one:
+            backup_options: '--an-option'
+            restor_options: '-xyz --another'
+        connection_two:
+            backup_options: '--a-first-one --a-second-one'
+            restor_options: '-O sample-value'
+```
+</div>
+<div class="standalone">
+
+Here is an example that sets options for all connections:
+
+```yml
+# db_tools.yaml
+backup_options: '--an-option'
+restore_options: '--a-first-one --a-second-one'
+```
+
+Or set a specific value list for each connection:
+
+```yml
+# db_tools.yaml
+connections:
+    connection_one:
+        backup_options: '--an-option'
+        restor_options: '-xyz --another'
+    connection_two:
+        backup_options: '--a-first-one --a-second-one'
+        restor_options: '-O sample-value'
+```
+</div>
+
+If you do not define your own default options, the following ones will be used
+according to the database vendor:
+
+* When backing up:
+  * MariaDB: `--no-tablespaces`
+  * MySQL: `--no-tablespaces`
+  * PostgreSQL: `-Z 5 --lock-wait-timeout=120`
+  * SQLite: `-bail`
+* When restoring:
+  * MariaDB: None
+  * MySQL: None
+  * PostgreSQL: `-j 2 --clean --if-exists --disable-triggers`
+  * SQLite: None
+
+## Anonymizer paths
+
+<div class="symfony">
+
+By default, the *DbToolsBundle* will look for custom *anonymizers* in 2 directories:
+
+* `%kernel.project_dir%/vendor/makinacorpus/db-tools-bundle/src/Anonymizer`
+* `%kernel.project_dir%/src/Anonymizer`
+
+If you want to put custom anonymizers in another directory or if you want to load
+a pack of anonymizers from an external library, you can modify/add paths:
+
+
+```yml
+# config/packages/db_tools.yaml
+db_tools:
+    anonymizer_paths:
+        - '%kernel.project_dir%/src/Anonymizer'
+        - '%kernel.project_dir%/vendor/myAnonymizerProvider/anonymizers/src'
+        # ...
+```
+</div>
+<div class="standalone">
+
+By default, the *DbToolsBundle* when used as a standalone CLI tool will not lookup for
+custom *anonymizers*.
+
+If you want to write custom anonymizers, in order for them to be found, you must specify
+paths where the source code lies:
```

Comment:
```suggestion
paths where the source code lives:
```

Reply:
This is valid sentences using the verb "to lie", but if you prefer I can change it ?

Reply:
ah ok, my bad, keep it